src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,97 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_INCREASE = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private int defaultLoad;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
+        this.defaultLoad = (int) (LOAD_FACTOR * DEFAULT_CAPACITY);
```

Comment:
you can remove brackets here: this.defaultLoad = (int) LOAD_FACTOR * DEFAULT_CAPACITY;